Source/WSTagsField.swift

```diff
@@ -326,6 +326,10 @@ open class WSTagsField: UIScrollView {
         // So always attempt to resign without checking.
         self.textField.resignFirstResponder()
     }
+    
+    open func reloadTextFieldInputViews() {
+        self.textField.reloadInputViews()
```

Comment:
What do you think instead adding a new method which will confuse lib users and override the current one:

```
override func reloadTextFieldInputViews() {
    // not calling `super.reloadTextFieldInputViews` but use the `textField` instead
    textField.reloadInputViews()
}
```

Reply:
Did you mean like this:
`    override open func reloadInputViews() {
        self.textField.reloadInputViews()
    }`

If so, I would have thought this would work and would even be a better solution! But for some reason, it doesn't seem to override correctly and it seems to call the superclass method still, causing the keyboard to hide. 🤔

I can't figure out why that would be happening!

Reply:
@jamesyorke Yes, exactly. I forgot to rename the method to `reloadInputViews`.

This will work, 100% sure:

```
open override func reloadInputViews() {
    self.textField.reloadInputViews()
}
```

^ This doesn't call the super class method.

Reply:
I tried this and it works as expected.

Reply:
It does work, as you said! Apologies, I did something incorrectly the first time I tried it. 

I have updated the Pull Request and tested it! 👍